public/js/widget_config.js

```diff
@@ -95,15 +95,19 @@ const createWidget = function (objWidget, objSocket) { // eslint-disable-line no
   const widgetContainer = jQuery(
     `<div class="widget ${widgetTypeUpper}" id="${objWidget.label}"></div>`
   )
-  const widgetHeader = '<div class="widget-header">' + objWidget.label + '</div>'
+  const widgetHeader = jQuery(
+    '<div class="widget-header">' + objWidget.label + '</div>'
+  )
   const widgetKebobMenu = jQuery(
-    '<img class="kebobMenu" src="img/kebobMenu.png"/>'
+    '<img class="widget-kebobMenu" src="img/kebobMenu.png"/>'
+  )
+  const widgetContent = jQuery(
+    '<div class="widget-content"></div>'
   )
-  const widgetContent = '<div class="widget-content"></div>'
 
   widgetKebobMenu.appendTo(widgetContainer)
-  jQuery(widgetHeader).appendTo(widgetContainer)
-  jQuery(widgetContent).appendTo(widgetContainer)
+  widgetHeader.appendTo(widgetContainer)
+  widgetContent.appendTo(widgetContainer)
```

Comment:
I'm not sure if there was a reason for having some of these as jQuery objects and some as strings, but to add the double-click I found it useful to just define the widgetHeader as a jQuery object.